### ISSUE TITLE
Specto not auto-updating library folders

### ISSUE THREAD

**Issue by metaltermite** [mrknow/filmkodi#818]
## Please follow the guide below / Podążaj za przewodnikiem
- Put an "x" into all boxes [ ] relevant to your *issue* (Wstaw "x" w każde pole [ ], które dotyczy twojego zgłoszenia)
- Use *Preview* tab to see how your issue will actually look like (Użyj *Perview* aby sprawdzić, czy wszystko wygląda dobrze)

---
## Kodi & OS
- Kodi verison: 17.1 Git:20170320-fc1619b
- OS version: Windows 10, 64-bit

---
## Addon / Wtyczka
- [ ] Filmy online - Mrknow
- [ ] MrknowTV
- [ x] Specto
- [ ] FanFILM
- Version / Wersja: 2018.03.20.1

---
## Issue / Zgłoszenie
- [ x] Bug report / Błąd
- [ ] Site request / Prośba o nową stronę
- [ ] Feature request / Prośba o nową funkcję
- [ ] Question / Pytanie
- [ ] Other / Inne

---
## Descryption / Opis

TV shows that were added to the library via Specto have stopped auto-updating. I have to manually auto-update the folders via the TOOLS menu in Specto to get new TV show episodes added to my library.

**Comment by dkma2547** [comment]
I am getting this issue too. Also my progress folder is not auto updating to show the next episode even though my trakt is updated in the website.

**Comment by JackDinn** [comment]
yea im getting this now too, was fine last week but not a single new episode has shown since about 3-4 days ago, although the episodes are there on specto if i search them.

**Comment by metaltermite** [comment]
It's weird because a few shows are auto-updating but others are not.

I've noticed if I search for a TV Show that I have added to my library via Specto, but isn't auto updating, _Add-ons > Specto > TV shows > Search_, the result will return the show as watched. However, if you drill down into the seasons view, it properly shows the latest season and subsequently the latest epises as unwatched.

So basically, maybe Specto is not auto-updating because Trakt is telling it that all episodes are watched?

**Comment by marcianna74** [comment]
When I request the program for that week, the links come up but it says no streams found. for oll of them even if there is a list of over 100 links.  And this is for all programs that I request.  Please help!  I used to like Specto but now, don't know what to think.
